### PR TITLE
Update framework (and other dependencies)

### PR DESCRIPTION
**Pull request by peppy**

### CODE REVIEW

**Review by smoogipoo — DISMISSED** [review]

File: osu.Game/Database/RealmAccess.cs [hunk 1]
```diff
@@ -902,7 +902,14 @@ private void createBackup(string backupFilename)
                 {
                     using (var source = storage.GetStream(Filename, mode: FileMode.Open))
                     using (var destination = storage.GetStream(backupFilename, FileAccess.Write, FileMode.CreateNew))
+                    {
+                        // source may not exist.
+                        if (source == null)
+                            return;
```
Comment (smoogipoo) [inline osu.Game/Database/RealmAccess.cs:908]: Any reason to still be creating the backup if the source doesn't exist?

Comment (peppy) [reply osu.Game/Database/RealmAccess.cs:908]: It doesn't create it due to the early `return`. Would you want a precondition check for file existing?

Comment (smoogipoo) [reply osu.Game/Database/RealmAccess.cs:908]: Am I missing something? I just tested in an isolated app:
```csharp
using (var fs = new FileStream("/home/smgi/Desktop/wangs.txt", FileMode.CreateNew, FileAccess.Write))
    return;
```
And it creates the file.

Comment (peppy) [reply osu.Game/Database/RealmAccess.cs:908]: Ah I get you now.

**Review by smoogipoo — APPROVED** [review]

**Pull request merged**